src/pyird/spec/continuum.py

```diff
@@ -87,10 +87,20 @@ def f(scale):
         df_tmp = pd.DataFrame(data,columns=['wav','order','flux','flat','continuum'])
         df_continuum = pd.concat([df_continuum,df_tmp],ignore_index=True)
     df_continuum['nflux'] = df_continuum['flux']/df_continuum['continuum']
-    df_continuum['nflat'] = df_continuum['flat']/df_continuum['continuum']
```

Comment:
Could you not delete this? (This seems to be set for the output of normalized flat spectrum at l. 579 in irdstream.py.)

Reply:
Sorry about that! It seems to be deleted accidentally. I will add it back!